ci-operator/step-registry/single-node/recert/single-node-recert-commands.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+recert_script=$(cat << 'EOF'
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function start_containers {
+  systemctl start crio
+  systemctl start kubelet
+}
+
+function stop_containers {
+  systemctl stop kubelet
+  crictl ps -q | xargs crictl stop || true
+  systemctl stop crio
+}
+
+function recert {
+  RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.13.6-x86_64
+  ETCD_IMAGE="$(oc adm release extract --from="$RELEASE_IMAGE" --file=image-references | jq '.spec.tags[] | select(.name == "etcd").from.name' -r)"
+  RECERT_IMAGE="quay.io/otuchfel/recert:latest"
+  podman run --authfile=/var/lib/kubelet/config.json --name recert_etcd --detach --rm --network=host --privileged --entrypoint etcd -v /var/lib/etcd:/store "${ETCD_IMAGE}" --name editor --data-dir /store
+  sleep 10 # TODO: wait for etcd
+  podman run -it --network=host --privileged -v /etc/kubernetes:/kubernetes -v /var/lib/kubelet:/kubelet -v /etc/machine-config-daemon:/machine-config-daemon ${RECERT_IMAGE} \
+      --etcd-endpoint localhost:2379 \
+      --static-dir /kubernetes \
+      --static-dir /kubelet \
+      --static-dir /machine-config-daemon \
+
+  podman kill recert_etcd
+
+  # workaround until https://github.com/omertuc/recert/blob/4d41d451ba57fbc9fd75781684906360696f384a/README.md?plain=1#L24 is resolved
+  rm -rf "/etc/machine-config-daemon/currentconfig"
+  touch "/run/machine-config-daemon-force"
+}
+
+stop_containers
+recert
+start_containers
+
+systemctl disable recert.service
+
+echo "Recert run successfully."
+EOF
+)
+
+# Base64 encode the script for use in the MachineConfig.
+b64_script=$(echo "${recert_script}" | base64 -w 0)
+
+machineconfig=$(oc create -f - -o jsonpath='{.metadata.name}' << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-recert
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          compression: gzip
```

Comment:
I believe the latest error is caused by this

Reply:
Thanks for spotting this! That's true, I `base64`-encoded the content but I didn't compress it. I believe compression is not important at this step, so I will just remove this line.